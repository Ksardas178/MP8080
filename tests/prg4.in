	MVI A b11111111
	LXI H 9 0	//addr -> HL	00000000|00001001
>L1
	MOV M A		//A -> M		addr <- A
	INX H		//HL++ (DCX?)
	JMP L1
//cells 9-32?