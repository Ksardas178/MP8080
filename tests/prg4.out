stored label L1 on line 5

[32;1mInput is correct
[0m
[30;1mCode analysis results:
[0m______________________________
00000000: 00111110
00000001: 11111111
00000010: 00100001
00000011: 00001001
00000100: 00000000
00000101: 01110111
00000110: 00100011
00000111: 11000011
00001000: 00000101
00001001: 00000000
